MapboxCoreNavigation/MMEEventsManager.swift

```diff
@@ -0,0 +1,241 @@
+import Polyline
+import MapboxDirections
+import AVFoundation
+import MapboxMobileEvents
+
+let SecondsForCollectionAfterFeedbackEvent: TimeInterval = 20
+
+struct DefaultEventDictionary {
+    var originalRequestIdentifier: String?
+    var requestIdentifier: String?
+    var lat: CLLocationDegrees?
+    var lng: CLLocationDegrees?
+    var originalGeometry: String?
```

Comment:
This property should be of type `Polyline`; rely on the conversion method to encode it into a string.